libs/scully/src/lib/utils/httpGetJson.ts

```diff
@@ -44,15 +44,23 @@ You can ignore the warning (TLS) or run scully with --no-warning
     };
     httpGet(opt, res => {
       const { statusCode } = res;
-      const contentType = res.headers['content-type'];
+      const responseContentType = res.headers['content-type'];
```

Comment:
I like this, but there is a fair chance that the return type is different from the content-type you are posting.
Perhaps we should add a bit of documentation, and give it the name `accepts-content-type`?

Reply:
I agree, this could be misleading in some situations. Changing it to `accepts-content-type` with good docs should definitely clarify the case.

Reply:
@SanderElias I tweaked the idea a little bit - I've changed it to `expectedContentType`. However I have mixed feelings about including it in the `HeadersObject`. Maybe it would be more appropriate to declare `headersOptions` property along headers property in `RouteTypeJson` object. This way we wouldn't mix actual headers with some custom options 🤔

Reply:
That is more in line with what I would do to solve it "properly", but we need to find a way to put it in without braking existing code, and still keep a sane API on httpGetJson.